app/src/main/java/com/example/sharewhatyoucanproject/utils/NotificationUtils.kt

```diff
@@ -4,15 +4,15 @@ import android.app.NotificationChannel
 import android.app.NotificationManager
 import android.content.Context
 import android.os.Build
-import com.example.sharewhatyoucanproject.DonorDrawerActivity
+import com.example.sharewhatyoucanproject.MainActivity
 import com.google.firebase.messaging.FirebaseMessaging
 
 fun createNotificationChannel(context: Context, userId:String,) {
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
         val channel =
             NotificationChannel(
-                DonorDrawerActivity.DonorConstants.CHANNEL_1_ID,
-                DonorDrawerActivity.DonorConstants.CHANNEL_2_ID,
+                MainActivity.DonorConstants.CHANNEL_1_ID,
```

Comment:
nit: you could make a separate object to contain all your constants so that you do not have to import `MainActivity` here, eg:

```
object Constants {
    CHANNEL_ID_<DESCRIPTION> = //
    CHANNEL_ID_<DESCRIPTION> = //
}
```

usage:
```
val channelId = Constants.CHANNEL_ID_<DESCRIPTION>
```

Reply:
done